pkg/statistics/component.go

```diff
@@ -45,6 +46,10 @@ func NewComponent(db keycloakb.EventsDBModule, keycloakClient KeycloakClient, lo
 	}
 }
 
+const (
+	RegExpMax = `^\d{1,2}$`
```

Comment:
Could be lower case if only used here... but I think it would be better to move this constant in api/statistics/api.go with other RegExp constants... something like RegExpTwoDigitsNumber or RegExpSmallNumber ?

Reply:
My mistake, this has been corrected